core/src/server/http/http_cached_date.cpp

```diff
@@ -0,0 +1,48 @@
+#include <server/http/http_cached_date.hpp>
+
+#include <chrono>
+#include <cstring>
+
+#include <userver/utils/datetime/wall_coarse_clock.hpp>
+
+#include <cctz/time_zone.h>
+
+USERVER_NAMESPACE_BEGIN
+
+namespace server::http {
+
+std::string_view GetCachedHttpDate() {
+  static thread_local std::chrono::seconds::rep last_second = 0;
+  static thread_local char last_time_string[128]{};
```

Comment:
Just std::string. In that case there's noo need in bytes copy and no need in result view

Reply:
Well, it will be one extra hop in memory, won't it?

Reply:
Oh, okay. Leave it as is :)